ctapipe/instrument/subarray.py

```diff
@@ -196,8 +201,9 @@ def tel_ids_to_indices(self, tel_ids):
         np.array:
             array of corresponding tel indices
         """
-        tel_ids = np.array(tel_ids, dtype=int, copy=False).ravel()
-        return self.tel_index_array[tel_ids]
+        tel_ids = np.array(tel_ids, dtype=int, copy=False)
+        shape = tel_ids.shape
+        return self.tel_index_array[tel_ids.ravel()].reshape(shape)
```

Comment:
Is this reshape fixing some other bug?

Reply:
Ah, no, I didn't want to add this here, was by mistake